ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -32,7 +32,10 @@ export BITSTREAM="--offline --list ${SHA}"
 # in case we've crashed the UART handler on the CW310's SAM3U
 trap 'ci/bazelisk.sh run //sw/host/opentitantool -- --interface=cw310 fpga reset' EXIT
 
-cw310_tags=( "cw310_test_rom" "cw310_rom" )
+# Loading the same bitstream onto the FPGA is fast, but loading a different
+# bitstream is slow. To minimize the number of bitstream transitions the FPGA
+# experiences, group tests by bitstream and invoke Bazel once for each group.
```

Comment:
nit: We don't really load the same bitstream, we skip this step if the FPGA is already programmed with the correct bitstream.

```suggestion
# Loading a bitstream onto the FPGA is slow. Group tests by bitstream and invoke Bazel once for each group to minimize the time spent on bitstream loading.
```

Reply:
Works for me :) Done.